dev/doc/changes.md

```diff
@@ -1,3 +1,10 @@
+## Changes beyond Coq 8.8
+
+### ML API
+
+Types Context.Rel.*.t are now in Constr.Context.Rel.*.t, so extra
```

Comment:
This is a markdown file so if you don't put backquotes around this, the star is interpreted as marking the begin of an emphasis.

Reply:
Thanks for noticing. Fixed.